modules/rest-util/test/blaze/middleware/fhir/output_test.clj

```diff
@@ -56,154 +78,143 @@
 
   (testing "_format overrides"
-    (are [accept format content-type]
-         (given (call resource-handler-200
-                      {:headers {"accept" accept}
-                       :query-params {"_format" format}})
-           :status := 200
-           [:headers "Content-Type"] := content-type
-           [:body parse-xml] := {:fhir/type :fhir/Patient :id "0"})
-      "application/fhir+json"
-      "application/fhir+xml"
-      "application/fhir+xml;charset=utf-8"
-
-      "application/fhir+json"
-      "application/xml"
-      "application/xml;charset=utf-8"
-
-      "application/fhir+json"
-      "text/xml"
-      "text/xml;charset=utf-8"
-
-      "application/fhir+json"
-      "xml"
-      "application/fhir+xml;charset=utf-8"
-
-      "*/*"
-      "application/fhir+xml"
-      "application/fhir+xml;charset=utf-8"
-
-      "*/*"
-      "application/xml"
-      "application/xml;charset=utf-8"
-
-      "*/*"
-      "text/xml"
-      "text/xml;charset=utf-8"
-
-      "*/*"
-      "xml"
-      "application/fhir+xml;charset=utf-8"))
+    (doseq [[accept format content-type] [["application/fhir+json" "application/fhir+xml" "application/fhir+xml;charset=utf-8"]
+                                          ["application/fhir+json" "application/xml" "application/xml;charset=utf-8"]
+                                          ["application/fhir+json" "text/xml" "text/xml;charset=utf-8"]
+                                          ["application/fhir+json" "xml" "application/fhir+xml;charset=utf-8"]
+                                          ["*/*" "application/fhir+xml" "application/fhir+xml;charset=utf-8"]
+                                          ["*/*" "application/xml" "application/xml;charset=utf-8"]
+                                          ["*/*" "text/xml" "text/xml;charset=utf-8"]
+                                          ["*/*" "xml" "application/fhir+xml;charset=utf-8"]]]
+      (given (call resource-handler-200
+                   {:headers {"accept" accept}
+                    :query-params {"_format" format}})
+        :status := 200
+        [:headers "Content-Type"] := content-type
+        [:body parse-xml] := {:fhir/type :fhir/Patient :id "0"})))
 
   (testing "not modified"
     (given (call resource-handler-304 {:headers {"accept" "application/fhir+xml"}})
+      :status := 304
       [:headers "Content-Type"] := nil
       :body := nil))
 
   (testing "failing XML emit"
     (given (call (special-resource-handler {:fhir/type :fhir/Patient :id "0" :gender #fhir/code"foo\u001Ebar"}) {:headers {"accept" "application/fhir+xml"}})
+      :status := 500
       [:headers "Content-Type"] := "application/fhir+xml;charset=utf-8"
+      [:body parse-xml :fhir/type] := :fhir/OperationOutcome
       [:body parse-xml :issue 0 :diagnostics] := "Invalid white space character (0x1e) in text to output (in xml 1.1, could output as a character entity)")))
 
+(deftest binary-resource-test
+
+  (testing "possible accept headers"
```

Comment:
I would say "returning the resource". "possible accept headers" is just a copy from above. But here we have to differentiate between returning the resource and returning the data.